### ISSUE TITLE
Default channel is not configured correctly.

### ISSUE THREAD

**Issue by sandersn80** [conda-forge/miniforge#279]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

When a .condarc/condrc file with:
```
channels:
-defaults
```
is still available within the home directory/user files (e.g. because they used Anaconda before), Miniforge will also download package from the Anaconda repository. This is because the defaults channels still seem to point to the commercial repositories instead of conda-forge (default from conda package?). For our central installations, we overruled this behavior by specifying that the defaults channel should be conda-forge:
```
default_channels:
- https://conda.anaconda.org/conda-forge
```
It would be very helpful if this could also be implemented in the .condarc of the Miniforge installation. Is that possible. Now it only contains:
```
channels:
- conda-forge
```
Some of our users manually downloaded the Miniforge installation package (e.g. to install Miniforge on Windows). If there are still .condarc/condarc files left on their system, they could still be using the commercial repositories by mistake. Especially with the changes Terms and Conditions of Anaconda in mind I think this would be a useful change. Or are do you expect this is a bug instead?

### Installed packages

```shell
n/a
```

### Environment info

```shell
Without overruling defaults channel:
     active environment : None
       user config file : /home/user/.condarc
 populated config files : /path/to/python3/3.8-miniforge/.condarc
                          /home/user/.condarc
          conda version : 4.12.0
    conda-build version : not installed
         python version : 3.8.12.final.0
       virtual packages : __linux=3.10.0=0
                          __glibc=2.17=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /path/to/python3/3.8-miniforge  (read only)
      conda av data dir : /path/to/python3/3.8-miniforge/etc/conda
  conda av metadata url : None
          channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /cadappl/python3/3.8-miniforge/pkgs
                          /home/user/.conda/pkgs
       envs directories : /home/user/.conda/envs
                          /path/to/python3/3.8-miniforge/envs
               platform : linux-64
             user-agent : conda/4.12.0 requests/2.27.1 CPython/3.8.12 Linux/3.10.0-1160.36.2.el7.x86_64 scientific/7.9 glibc/2.17
                UID:GID : 1106:1000
             netrc file : None
           offline mode : False

Expected/with altered defaults channel:
     active environment : None
       user config file : /home/user/.condarc
 populated config files : /path/to/python3/3.8-miniforge/.condarc
                          /home/user/.condarc
          conda version : 4.12.0
    conda-build version : not installed
         python version : 3.8.12.final.0
       virtual packages : __linux=3.10.0=0
                          __glibc=2.17=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /path/to/python3/3.8-miniforge  (read only)
      conda av data dir : /path/to/python3/3.8-miniforge/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /cadappl/python3/3.8-miniforge/pkgs
                          /home/user/.conda/pkgs
       envs directories : /home/user/.conda/envs
                          /path/to/python3/3.8-miniforge/envs
               platform : linux-64
             user-agent : conda/4.12.0 requests/2.27.1 CPython/3.8.12 Linux/3.10.0-1160.36.2.el7.x86_64 scientific/7.9 glibc/2.17
                UID:GID : 1106:1000
             netrc file : None
           offline mode : False
```

**Comment by sandersn80** [comment]
Any news regarding this change request? I think I would prevent a lot of confusion....

**Comment by hmaarrfk** [comment]
FWIW, i've seen the issue you raised, but it does invoke a rather large change in behavior.

I'm not so sure we are ready for this. But maybe we are?

Still trying to formulate my thoughts.

**Comment by isuruf** [comment]
No. It's not desirable. `defaults` channel (i.e. pkgs/main + pkgs/r) will not be accessible using `-c defaults` if we do that.

**Comment by sandersn80** [comment]
Hi @hmaarrfk @isuruf I understand your doubts. I though the main and r channels could still be accessed trough anaconda.org? These also would require a license for large companies, but in that case users specifically have to enable those channels. In the current situation users are unaware of the usage of the commercial repos if there are still conda/.condarc files on their account or system that enabled the defaults channel. This could specifically be an issue for commercial usage.

**Comment by isuruf** [comment]
So you want to do a backward incompatible change to benefit a certain group of users while breaking another group of users' workflows?

**Comment by sandersn80** [comment]
Hi @isuruf, No, that is not my intension, My apologies. But in the README,md the following is mentioned:

-  **conda-forge set as the default (and only) channel.**
- Packages in the base environment are obtained from the [conda-forge channel](https://anaconda.org/conda-forge).

A lot of users are currently not aware that if they uninstall Anaconda (which often creates a .condarc with the defaults channel enabled) and do not cleanup there.condarc files (which could also be located in a lot of locations) the anaconda.com/pkgs repos are still used. If this is still preferred, I would add a notification that the usage of the defaults channel would still enable Miniforge to use the commercial channels. The README.md seems to imply that switching to Miniforge only uses conda-forge.

**Comment by isuruf** [comment]
Would you be able to send a PR to update the README.md to make this clarification?

Maybe also open an issue in https://github.com/conda/conda about channels being aggregated instead of overriden from user configs?

**Comment by sandersn80** [comment]
Thank you. I will do so.

**Comment by isuruf** [comment]
There's actually an option https://github.com/conda/conda/blob/efd28dc73541ab470d1ee72b165b934fc492abec/docs/source/user-guide/configuration/use-condarc.rst#allow-other-channels-allow_other_channels

**Comment by sandersn80** [comment]
> There's actually an option https://github.com/conda/conda/blob/efd28dc73541ab470d1ee72b165b934fc492abec/docs/source/user-guide/configuration/use-condarc.rst#allow-other-channels-allow_other_channels

Thank you. For our central installation we indeed disabled the usage of other channels (and whitelisted the channels that users are allowed to use). But there are also users who download the installation themselves, so for that we are out of control. 

You also mentioned:
"Maybe also open an issue in https://github.com/conda/conda about channels being aggregated instead of overriden from user configs?". Could you further clarify what you exactly suggest?
